README.md

```diff
@@ -1,72 +1,96 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+https://drive.google.com/file/d/12lGLaslYanPiUrMC63FQ3_QKaRoJ3_mG/view?usp=sharing
 
-# Project Name
+#### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./MIT.md) of the design.
```

Comment:
[OPTIONAL ] : you can make your readme file look more professional by removing this line since you already have it in the acknowledgments section 👍 😄